src/stream.js

```diff
@@ -101,8 +101,16 @@ export default class Stream {
    * @see http://nodejs.org/api/stream.html#stream_readable_pipe_destination_options
    */
   pipe(destination) {
-    this.on('data', function(data) {
-      destination.push(data);
+    this.on('data', function() {
```

Comment:
Could this be replaced by:
```js
this.on('data', (...args) => {
  args.forEach(arg => destination.push(arg));
});
```
?

Reply:
possibly, but then we create an array object even if we only have one argument and the comment in `trigger` seems to imply that there is a performance implication to doing that.